aoe4/handlers.js

```diff
@@ -29,6 +29,8 @@ async function getPlayerWinRate(player, opponent, thresholdHours) {
   var lastgame = games[0] ? Date.parse(games[0].started_at) : Date.now();
 
   for (const game of games) {
+    if (!game.duration)
```

Comment:
Move it below line 44, that handles the rolling window. That way it'll trigger the rolling window, but not count in the statistics.